experimental/origin-dapp2/src/pages/_ListingBadge.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+const ListingBadge = ({ status, featured }) => {
+  if (status === 'sold') {
+    return <div className="badge badge--sold">sold</div>
+  }
+  // TODO: Unavailable (multi unit, pending)
```

Comment:
Should I add multi unit support to GraphQL in this PR, or a separate one? I can add it all in this PR, but just wondering.

Reply:
Lets do that in a separate PR...

Reply:
Ok, created #1162 for that.